core/src/avm2/globals/xml.rs

```diff
@@ -127,28 +127,30 @@ pub fn attribute<'gc>(
     let this = this.unwrap();
     let xml = this.as_xml_object().unwrap();
     let name = args[0];
-    let qname = match name {
-        Value::String(s) => QName::new(activation.avm2().public_namespace, s),
+    let multiname = match name {
+        Value::String(s) => QName::new(activation.avm2().public_namespace, s).into(),
```

Comment:
Note: all these uses of `QName::new(...).into()` could be simply converted to `Multiname::new(...)`